content/en/docs/concepts/security/overview.md

```diff
@@ -0,0 +1,151 @@
+Network access to Nodes (Worker Servers) | Nodes should be configured to _only_ accept connections (via network access control lists) from the masters on the specified ports, and accept connections for services in Kubernetes of type `NodePort` and `LoadBalancer`. If possible, this nodes should not exposed on the public internet entirely.
+Kubernetes access to Cloud Provider API | Each cloud provider will need to grant a different set of permissions to the Kubernetes Masters and Nodes, so this recommendation will be more generic. It is best to provide the cluster with cloud provider access that follows the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege) for the resources it needs to administer. An example for Kops in AWS can be found here: https://github.com/kubernetes/kops/blob/master/docs/iam_roles.md#iam-roles
+Access to etcd | Access to etcd (the datastore of Kubernetes) should be limited to the masters only. Depending on your configuration you should also attempt to use etcd over TLS. More info can be found here: https://github.com/etcd-io/etcd/tree/master/Documentation#security
+etcd Encryption | Wherever possible it's a good practice to encrypt all drives at rest, but since etcd holds the state of the entire cluster (including Secrets) its disk should especially be encrypted at rest.
+
+## Cluster
+
+This section will provide links to more exploration for securing the aspects of
+workloads in Kubernetes. There are essentially two areas of concern for securing
+Kubernetes:
+
+* Securing the components that are configurable which _make up_ the cluster
+* Securing the components which run _in_ the cluster
+
+
+### Components _of_ the Cluster
+For more information on securing the components of the cluster, go [here](/docs/tasks/administer-cluster/securing-a-cluster).
+
+### Components _in_ the Cluster (your application)
+Depending on the attack surface of your application, you may want to focus on specific
+aspects of security. For example, if you are running a service (Service A) that is critical
+in a chain of other resources and a separate workload (Service B) which is
+vulnerable to a resource exhaustion attack, by not putting resource limits on
+Service B you run the risk of also compromising Service A. Below is a table of
+links of things to consider when securing workloads running in Kubernetes.
+
+Area of Concern for Workload Security | Recommendation |
+------------------------------ | ------------ |
+RBAC Authorization (Access to the Kubernetes API) | https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+Authentication | https://kubernetes.io/docs/reference/access-authn-authz/controlling-access/
+Application secrets management (and encrypting them in etcd at rest) | https://kubernetes.io/docs/concepts/configuration/secret/ <br> https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/
+Pod Security Policies | https://kubernetes.io/docs/concepts/policy/pod-security-policy/
```

Comment:
- Again, etcd is specifically called out
- Clean up the `<br`?
- Do you need to write out the links, as they are internal to the k8s docs?

Reply:
etcd is the brains of Kubernetes. You must use it in order to use Kubernetes. The `<br>` tag is there to force hugo's BlackFriday markdown engine to separate the links so they are more readable to an end user